packages/app/obojobo-document-engine/package.json

```diff
@@ -61,9 +61,11 @@
 		"smoothscroll-polyfill": "^0.4.4",
 		"sysend": "^1.3.3",
 		"underscore": "1.10.2",
-		"url-parse": "^1.4.7 "
+		"url-parse": "^1.4.7 ",
+		"uuid": "^8.3.2"
```

Comment:
You should be able to use our internal uuid library instead, `import uuid from 'obojobo-document-engine/src/scripts/common/util/uuid'`